unattended_installer/install_functions/opendistro/checks.sh

```diff
@@ -147,6 +158,38 @@ function checkArguments() {
 
 }
 
+function checkTools() {
+
+    # -------------- Check tools required to run the script (awk, sed, etc.) -----------------------------------------
+
+    toolList=(  "awk" "cat" "chown" "cp" "curl" "echo" "export"
+                "free" "grep" "kill" "mkdir" "mv" "rm" "sed"
+                "sudo" "tar" "touch" "uname")
+
+    missingtoolsList=()
+    for command in "${toolList[@]}"
+    do
+        if [ -z "$(command -v $command)" ]; then
+            missingtoolsList+=($command)
+            missingtoolsStatus="true"
+        fi
+    done
+
+    if [ -n "${missingtoolsStatus}" ]; then
+
+        logger "---------------------------------- Missing tool -----------------------------------"
+        logger "Missing tool report:"
+        for tool in "${missingtoolsList[@]}"; do
+            if [ -n "$($command -L -n | grep DROP | grep $port)" ]; then
+                logger "                 ...$missingtoolsList is not installed. "
```

Comment:
Don't use multiline. One line for all tools separated by `,`.

Reply:
Done